web/src/Ports.js

```diff
@@ -0,0 +1,9 @@
+
+// This file is not being used yet
```

Comment:
When you do want to use this file you should be able to `require` it from `index-demo.ejs` and have it work.

Also on the todo list there is to separate out `index-demo.ejs` JS code into an `index-demo.js` file and then have WebPack inject this into the generated `index.html` file. See for an example:
https://github.com/swarmfund/sv-voting-mvp/blob/master/web/index.js
https://github.com/swarmfund/sv-voting-mvp/blob/master/web/index.ejs
(Notice that index.js isn't `require`d by `index.ejs` - webpack does the injection)